src/GHC/Util.hs

```diff
@@ -79,21 +69,18 @@ parseFileGhcLib filename str flags =
               if takeExtension filename /= ".lhs" then str else unlit filename str
     parseState = mkPState flags buffer location
 
+-- TODO(SF): GhclibParserEx.parsePragmasIntoDynFlags doesn't take
+-- enabled/disabled extensions into consideration yet.
```

Comment:
Isn't that a super important thing to do?

Reply:
Right - I will enhance the `ghc-lib-parser-ex` implementation so that we can replace this version. Just wanted to get `ghc-lib-parser-ex` seated first before new changes.

Reply:
Makes sense - sounds good.